kubernetes/staging/14-pod-disruption-budget.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: policy/v1
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)